Cargo.toml

```diff
@@ -96,7 +96,7 @@ sqlx = { version = "0.7.3", default-features = false, features = ["sqlite", "mac
 strum = { version = "0.26.2", features = ["derive"] }
 tempfile = "3"
 thiserror = "1.0"
-tokio = { version = "1.37.0", features = ["rt-multi-thread", "macros"] }
+tokio = { version = "1.37.0", features = ["rt-multi-thread", "macros","io-util", "net", "time"] }
```

Comment:
I'd rather specify this additional features per-crate, than here globally.

Reply:
Done.